.github/dependabot.yml

```
@@ -9,3 +10,24 @@ updates:
     labels:
       - area/dependencies
       - area/ci
+  # keycloak UI dependencies
+  - package-ecosystem: npm
+    directory: /themes/src/main/resources/theme/keycloak/common/resources/
+    schedule:
+      interval: daily
+      time: "00:01"
+      timezone: UTC
+    open-pull-requests-limit: 6
+    labels:
+      - area/dependencies
+  # keycloak.v2 UI dependencies
+  - package-ecosystem: npm
+    directory: /themes/src/main/resources/theme/keycloak.v2/account/src
+    schedule:
+      interval: daily
+      time: "00:01"
+      timezone: UTC
+    open-pull-requests-limit: 6
+    labels:
+      - area/dependencies
```

The `area/account/ui ` label should be added here as well.
```suggestion
      - area/dependencies
      - area/account/ui 
```